Config/database/database.ts

```diff
@@ -93,11 +94,48 @@ const remapLine = (rawLine: any) => {
   return Line;
 }
 
+const query = async (searchQuery: string, searchType: number) => {
+  await initSchema();
+
+  return new Promise((resolve, reject) => {
+    let dbQuery = '';
+    let searchColumn = '';
+    let condition = '';
+
+    // Sanitize query
+    const saniQuery = searchQuery.trim().replace("'", "\\'");
+    const MAX_RESULTS = 10;
+    const resultsOrder = [];
+    switch (searchType) {
+      case consts.SEARCH_TYPES.FIRST_LETTERS: // First letter start
+      case consts.SEARCH_TYPES.FIRST_LETTERS_ANYWHERE: {
+        searchColumn = 'FirstLetterStr';
+        let operator = searchType === consts.SEARCH_TYPES.FIRST_LETTERS ? 'BEGINSWITH' : 'CONTAINS';
+        condition = `${searchColumn} ${operator} '${dbQuery}'`;
```

Comment:
for this to work we need these lines from desktop realm search: 
```javascript
 for (let x = 0, len = saniQuery.length; x < len; x += 1) {
          let charCode = saniQuery.charCodeAt(x);
          if (charCode < 100) {
            charCode = `0${charCode}`;
          }
          if (charCode === '042') {
            isWildChar = true;
            dbQuery += ',*';
            operator = 'LIKE';
          } else {
            dbQuery += `,${charCode}`;
          }
        }
```
(might need some work for it to work with typescript